src/Analysis/Engine/Impl/Interpreter/Ast/AstBuiltinsPythonModule.cs

```diff
@@ -119,6 +120,10 @@ public AstBuiltinsPythonModule(PythonLanguageVersion version)
                     if (typeId == BuiltinTypeId.Bool) {
                         boolType = m as IPythonType;
                     }
+
+                    if (typeId == BuiltinTypeId.NoneType) {
+                        noneType = m as IPythonType;
```

Comment:
Can it be something else than `IPythonType`?

Reply:
No, because the `AstPythonConstant` constructor accepts that specifically.

Reply:
Then let's do a direct cast: `noneType = (IPythonType)m`;

Reply:
Done, and some more improvements in the same vein.